docusaurus.config.ts

```diff
@@ -52,6 +52,7 @@ const config: Config = {
   ],
 
   themeConfig: {
+    image: 'https://docs.status.network/img/sn_logo.png',
```

Comment:
Why not just `/img/sn_logo.png`? 
It should point to the same resource this way and be safe in case of domain change if I'm not mistaken

Reply:
this is actually for OpenGraph `og:image` meta tag which [only supports absolute URLs](https://stackoverflow.com/questions/9858577/open-graph-can-resolve-relative-url)

Reply:
I think docusaurus will automatically resolve based on baseUrl + static folder, we can leave it like this but keep in mind if it acts up in the future